src/current/v24.3/create-function.md

```diff
@@ -294,77 +294,84 @@ The preceding example modifies a given `name` value and returns the `NEW` [trigg
 
 The following example defines a function using the `SECURITY DEFINER` clause. This causes the function to execute with the privileges of the function owner.
 
-Create two roles:
+1. Create two roles:
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-CREATE ROLE owner;
-CREATE ROLE invoker;
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE ROLE owner;
+    CREATE ROLE invoker;
+    ~~~
 
-Grant a [`SELECT` privilege]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on the `user_promo_codes` table to the `owner` role.
+1. Grant a [`SELECT` privilege]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on the `user_promo_codes` table to the `owner` role.
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-GRANT SELECT ON TABLE user_promo_codes TO owner;
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    GRANT SELECT ON TABLE user_promo_codes TO owner;
+    ~~~
 
-Set your role to `owner`.
+1. Set your role to `owner`.
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-SET ROLE owner;
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET ROLE owner;
+    ~~~
 
-Create a simple `SECURITY DEFINER` function that reads the contents of `user_promo_codes`.
+1. Create a simple `SECURITY DEFINER` function that reads the contents of `user_promo_codes`.
 
-{% include_cached copy-clipboard.html %}
-~~~ sql
-CREATE OR REPLACE FUNCTION get_codes() 
-  RETURNS SETOF RECORD 
-  LANGUAGE SQL 
-  SECURITY DEFINER
-  AS $$
-    SELECT * FROM user_promo_codes;
-  $$;
-~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE OR REPLACE FUNCTION get_codes() 
+      RETURNS SETOF RECORD 
+      LANGUAGE SQL 
+      SECURITY DEFINER
+      AS $$
+        SELECT * FROM user_promo_codes;
+      $$;
+    ~~~
 
-Set your role to `invoker`.
+1. Grant the [`EXECUTE` privilege]({% link {{ page.version.version }}/grant.md %}#supported-privileges) on the `get_codes` function to the `invoker` role.
```

Comment:
Since we're adding this detail, it might be nice to call out that it's not necessary for the public schema for the reason we talked about earlier.

Reply:
Added!